connector_salesforce/salesforce_contact/importer.py

```diff
@@ -0,0 +1,145 @@
+from ..unit.mapper import AddressMapper
+_logger = logging.getLogger('salesforce_connector_contact_import')
+
+
+@salesforce_backend
+class SalesforceContactImporter(SalesforceImportSynchronizer):
+    _model_name = 'connector.salesforce.contact'
+
+    def _before_import(self):
+        """Hook called before importing a contact
+        Ensure that related account is present
+        if not it is imported
+        """
+        assert self.salesforce_record
+        with self.session.change_context({'active_test': False}):
+            account_id = self.session.search(
+                'connector.salesforce.account',
+                [('salesforce_id', '=', self.salesforce_record['AccountId'])]
+            )
+        if not account_id:
+            import_record(
+                self.session,
+                'connector.salesforce.account',
+                self.backend_record.id,
+                self.salesforce_record['AccountId']
+            )
+
+
+@salesforce_backend
+class SalesforceDirectBatchContactImporter(SalesforceDirectBatchSynchronizer):
+    _model_name = 'connector.salesforce.contact'
+
+
+@salesforce_backend
+class SalesforceDelayedBatchContactImporter(
+        SalesforceDelayedBatchSynchronizer):
+    _model_name = 'connector.salesforce.contact'
+
+
+@salesforce_backend
+class SalesforceContactAdapter(SalesforceRestAdapter):
+    _model_name = 'connector.salesforce.contact'
+    _sf_type = 'Contact'
+
+
+@salesforce_backend
+class SalesforceContactMapper(AddressMapper):
+    _model_name = 'connector.salesforce.contact'
+
+    direct = [
+        ('MailingStreet', 'street'),
+        ('MailingPostalCode', 'zip'),
+        ('MailingCity', 'city'),
+        ('Fax', 'fax'),
+        ('Phone', 'phone'),
+        ('AssistantPhone', 'sf_assistant_phone'),
+        ('OtherPhone', 'sf_other_phone'),
+        ('MobilePhone', 'mobile'),
+        ('Title', 'function'),
+        ('Email', 'email')
+    ]
+
+    @only_create
+    @mapping
+    def backend_id(self, record):
+        return {'backend_id': self.backend_record.id}
+
+    @only_create
+    @mapping
+    def is_company(self, record):
+        return {'is_company': False}
+
+    @only_create
+    @mapping
+    def name(self, record):
+        compound_name = ' '.join((record['LastName'], record['FirstName']))
+        return {'name': compound_name}
+
+    @mapping
+    def country_id(self, record):
+        country_id = self._country_id(record, 'MailingCountryCode')
+        return {'country_id': country_id}
+
+    @mapping
+    def state_id(self, record):
+        state_id = self._state_id(record, 'MailingState', 'MailingCountryCode')
+        return {'state_id': state_id}
+
+    @mapping
+    def customer(self, record):
+        return {'customer': True}
+
+    @mapping
+    def active(self, record):
+        return {'active': True}
+
+    @mapping
+    def title_id(self, record):
+        title_id = self._title_id(record, 'Salutation')
```

Comment:
No it is the field name in Salesforce model. https://www.salesforce.com/developer/docs/api/Content/sforce_api_objects_account.htm